include/dwg.h

```diff
@@ -2339,6 +2339,7 @@ typedef struct _dwg_object_BLOCK_HEADER
   BITCODE_H layout;
   BITCODE_RS unknown_r11;
   BITCODE_RS unknown1_r11;
+  BITCODE_RC r11_unknown;
```

Comment:
Rather re-use the existing RS with FIELD_CAST (unknown_r11, RC, RS, 0); 
There's no overlap, and maybe they do mean the same